package-rfcx/rfcx/_api_rfcx.py

```diff
@@ -14,6 +14,12 @@ def streamSegments(token, stream_id, start, end, limit, offset):
     url = '{}{}?{}'.format(host, path, urllib.parse.urlencode(data, True))
     return _request(url, token=token)
 
+def annotations(token, start, end, classifications=None, stream_id=None, limit=1000, offset=0):
+    data = {'start': start, 'end': end, 'classifications[]': classifications, 'stream_id': stream_id, 'limit': limit, 'offset': offset}
```

Comment:
Stas added new parameters to this API last week.

Reply:
Should be `is_positive` and `is_something_else_that_i_cant_remember`

Reply:
From what have I check, it is increase an update on the `streams/:id/annotations` using `post` and it is for create an annotation belonging to a stream. I think it is not an endpoint to be on the SDK.

Reply:
Ok - we can talk about this.